Clear/Instr.lean

```diff
@@ -1,6 +1,7 @@
 import Mathlib.Data.Nat.Defs
 import Clear.UInt256
 
+set_option linter.dupNamespace false
```

Comment:
Would it be time consuming to fix the namespace X { def X.foo } issues?

Reply:
Hi, I have proposed a fix in https://github.com/NethermindEth/Clear/pull/5. See:
- [Rename `Instr` type to `GInstr` (for "General Instruction") - To avoid name clash with section `Clear.Instr`](https://github.com/NethermindEth/Clear/commit/abbd304a9ab37e24e9020d26ccce2fa09e64fd78)